tests/conftest.py

```diff
@@ -229,6 +229,12 @@ def empty_config(tmp_path) -> Config:
         configuration.CONFIG = current_config
         manager.load_profile(current_profile_name)
```

Comment:
by the way, if the goal is strictly to reconstruct the state before this fixture was called, then I guess in principle one should check whether the "current profile" was loaded before or not.

I see that the current `Manager` implementation does not provide information on whether that was actually the case.
It may make sense to add an
```python
@property
def is_profile_loaded(self):
  return self._profile is not None
```
property (could also be used inside the manager implementation in some places)